.eslintrc

```diff
@@ -61,5 +61,6 @@
         ]
       }
     }
-  }
+  },
+  "parser": "babel-eslint"
```

Comment:
I used optional chaining for part of the globe. Eslint didn't recognize it so I needed to use babel as the parser.